app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class InvitationsController < ApplicationController
+  before_action :require_user
+
+  def new
+    @invitation = Invitation.new
+  end
+
+  def create
+    @invitation = Invitation.new(invitation_params)
+    @invitation.author = current_user
```

Comment:
Alright. 

If only there was a nicer way of merging hashes then using the merge method.
Unfortunately, this doesn't work: :smile: 
```ruby
@invitation = Invitation.new(invitation_params, author: current_user )
```